test/unit/app/set-router.spec.ts

```
@@ -5,7 +5,7 @@ import {
     TrieRouter,
     defineCoreHandler,
 } from '../../../src';
-import type { Route, RouteEntry } from '../../../src';
+import type { Route, Handler } from '../../../src';
```

ESLint

[error] Member 'Handler' of the import declaration should be sorted alphabetically.

(sort-imports)